x-pack/plugins/secops/public/pages/network/translations.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const NETWORK = i18n.translate('xpack.secops.network', {
+  defaultMessage: 'Network',
+});
+
+export const NO_FILEBEAT_INDICES = i18n.translate('xpack.secops.network.noFilebeatIndicies', {
+  defaultMessage: "Looks like you don't have any Filebeat indices.",
+});
+
+export const KQL_PLACE_HOLDER = i18n.translate('xpack.secops.network.kqlPlaceHolder', {
+  defaultMessage: 'Search… (e.g. network.name:"foo" AND process.name:"bar")',
+});
+
+export const LETS_ADD_SOME = i18n.translate('xpack.secops.network.letsAddSome', {
+  defaultMessage: "Let's add some!",
+});
```

Comment:
Per the guidelines:
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types

I think these are suppose have a `Description` suffix at the end of the keys if not something else?